pkg/drivers/kic/oci/types.go

```diff
@@ -105,7 +106,16 @@ type Mount struct {
 // '[host-path:]container-path[:<options>]' The comma-delimited 'options' are
 // [rw|ro], [Z], [srhared|rslave|rprivate].
 func ParseMountString(spec string) (m Mount, err error) {
-	switch fields := strings.Split(spec, ":"); len(fields) {
+	f := strings.Split(spec, ":")
+	fields := f
+	if runtime.GOOS == "windows" {
```

Comment:
I'd feel much more comfortable with this if ParseMountString had a unit test to demonstrate the behavior.

Reply:
totally fair, I'll add one

Reply:
and added! it actually found a bug in the new code, so good call.